building/get-apt-signing-key.sh

```diff
@@ -0,0 +1,21 @@
+while read branch key; do
+	if [ "${branch}" == "${HOMEWORLD_APT_BRANCH}" ] || [ "${branch}" == "$(echo ${HOMEWORLD_APT_BRANCH} | cut -d '/' -f 1)" ]; then
+		HOMEWORLD_APT_SIGNING_KEY="${key}"
+		break
+	fi
+done < signing-keys
+
+if [ -z "${HOMEWORLD_APT_SIGNING_KEY:-}" ]
+then
+	echo 'apt branch not found in signing-keys'
```

Comment:
Since the result is normally echoed, it might be worth making sure that errors are sent to stderr? (`1>&2`)